### PR TITLE
PEP8 and pyflakes fixups for numpy.core

### PR DESCRIPTION
**Pull request by charris**
PEP8 and pyflakes fixups for numpy/core/*.py

### CODE REVIEW

File: numpy/core/numeric.py [hunk 1]
```diff
@@ -1281,7 +1278,8 @@ def tensordot(a, b, axes=2):
     bs = b.shape
     ndb = len(b.shape)
     equal = True
-    if (na != nb): equal = False
+    if (na != nb):
```
Comment (jaimefrio) [inline numpy/core/numeric.py]: Don't need the parenthesis here.

File: numpy/core/numerictypes.py [hunk 2]
```diff
@@ -334,15 +331,15 @@ def _add_aliases():
...
-# Integers handled so that
-# The int32, int64 types should agree exactly with
-#  PyArray_INT32, PyArray_INT64 in C
-# We need to enforce the same checking as is done
-#  in arrayobject.h where the order of getting a
-#  bit-width match is:
-#       long, longlong, int, short, char
-#   for int8, int16, int32, int64, int128
-
+# Integers handled so that The int32, int64 types should agree exactly with
```
Comment (jaimefrio) [inline numpy/core/numerictypes.py]: Periods missing and/or capitalization is all messed in this paragraph.

Comment (charris) [inline numpy/core/numerictypes.py]: Yeah, I had not quite got the comment figured out. I think it is OK now

**Comment by jaimefrio** [comment]
In it goes, thanks Chuck.

**Comment by charris** [comment]
@jaimefrio Thanks for the review. These long PRs can be tedious to read through.

**Comment by jaimefrio** [comment]
Yeah, not like you, having all the fun running pyflakes all over the place... ;-)

**Pull request merged**